.env

```diff
@@ -0,0 +1,10 @@
+# MySQL настройки
```

Comment:
only ever write comments in English
remove this with git rm --cached .env (it will keep the file locally)
add .env.template with actual values removed
add .env to .gitignore